testgrid/config.md

```diff
@@ -14,18 +14,18 @@ Testgrid is composed of:
 * A list of **dashboards**, or collections of dashboard tabs
 * A list of **dashboard groups** of related dashboards.
 
-Most of these objects are listed in [`config.yaml`]
+Most of these objects are simply listed in a [YAML config file][configuration] for Testgrid to consume.
 
 ## Prow Job Configuration
 
-If you just have a prowjob you want to appear in an existing dashboard, add annotations to that
-prowjob. You don't need to change a config file here.
+If you just have a [Prow job configuration](/config/jobs) you want to appear in an existing
```

Comment:
You're linking to the prow.k8s.io jobs here, but talking about prow in the abstract, it's unclear to me which of these should be changed to match.
```suggestion
If you just have a [Prow job configuration](/prow/jobs.md) you want to appear in an existing
```